app/mailers/mailings/devise.rb

```diff
@@ -5,18 +5,19 @@ class Devise < ApplicationMailer
     include ::Devise::Controllers::UrlHelpers
 
     def confirmation_instructions(record, token, _opts = {})
-      Rails.logger.debug(confirmation_url(record, confirmation_token: token)) if Rails.env.development?
+      url = new_member_confirmation_url(confirmation_token: token)
```

Comment:
This is an incorrect change, this path is used for confirming an email address change, not activating a new account. If the user clicks the button now, it will simply redirect the user to the Koala home page without confirming the new email address; this should remain along the lines of `url = confirmation_url(record, confirmation_token: token)`